webserver/templates/user/export.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html' %}
+{% block title %} Export for "{{ user.musicbrainz_id }}" - ListenBrainz {% endblock %}
+
+{% block content %}
+	<h2 class="page-title">{{ user.musicbrainz_id }}</h2>
+	{% if current_user.is_authenticated() and current_user == user %}
+		<h3>Export from ListenBrainz</h3>
+		<p>
+			<strong> Notes about the ListenBrainz export process : </strong>
+		</p>
+		<p>
+				Now you can export your listens to a file.
+				Just choose one of the export formats and let ListenBrainz do the hard work of figuring out what to do.
```

Comment:
We may be adding more formats in future, so i wrote a generic line. :)
Anyways, will change asap.